src/dashboard/pages/DashboardPage.vue

```diff
@@ -167,5 +188,14 @@ export default {
     });
     this.$store.dispatch('db/ensureActivities');
   },
+  async created() {
+    await this.$store.dispatch('db/ensureDonations');
+    this.distribute();
+  },
+  watch: {
+    creators() {
+      this.distribute();
```

Comment:
Why is this needed? Wouldn't this mean that any change to the distribution is reset if creators change somehow?

Reply:
Yes it would. Not really needed i guess, but all of that logic will be changed by @dwilid s PR #213

Reply:
Alright then